### PR TITLE
Modules cleanup

### PR DESCRIPTION
**Pull request by kravciak**
I created list of modules that might not be used by openqa anymore. It probably still contains some usefull code, need more opinions on that.

Is it desirable to have such cleanups?

Script (ignore dirty code) used to produce module list:
https://gitlab.suse.de/qa/scripts/blob/master/mkravec/openqa-modules-cleanup/run.bash

### CODE REVIEW

**Comment by coolo** [comment]
the sle11 tests are there for a reason - most of them are referenced by the main.pm in sle11 branch. Other tests of your findings are indeed old workarounds no longer required.

**Comment by kravciak** [comment]
Removed modules mentioned in sle11 & yast-modules main.pm. Those modules & branches are kept because of webui history or some other reason?

File: CLEANUP_MODULES [hunk 1]
```diff
@@ -0,0 +1,11 @@
+tests/console/ttylogin4.pm
+tests/console/yast2_nis.pm
+tests/installation/inst_mediacheck.pm
+tests/x11/mate_reconfigure_openqa.pm
```
Comment (coolo) [inline CLEANUP_MODULES]: the reconfigure_openqa are obsolete - we handle other desktops differently meanwhile

File: CLEANUP_MODULES [hunk 2]
```diff
@@ -0,0 +1,11 @@
+tests/console/ttylogin4.pm
```
Comment (coolo) [inline CLEANUP_MODULES]: ttylogin was a workaround before tests selected their consoles - obsolete

File: CLEANUP_MODULES [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+tests/console/yast2_nis.pm
+tests/installation/inst_mediacheck.pm
```
Comment (coolo) [inline CLEANUP_MODULES]: that's part of sle11 - not obsolete

File: CLEANUP_MODULES [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+tests/x11/mate_reconfigure_openqa.pm
+tests/x11/reboot_gnome_sle11.pm
+tests/x11/enlightenment_reconfigure_openqa.pm
+tests/x11/lxqt_reconfigure_openqa.pm
+tests/x11/awesome_reconfigure_openqa.pm
+tests/x11regressions/evolution/evolution_task_ews.pm
```
Comment (coolo) [inline CLEANUP_MODULES]: these wait for the test account to be reenabled - not obsolete

**Comment by okurz** [comment]
Can we please make sle11 a full branch including all test modules and delete them from master?

**Comment by okurz** [comment]
please rebase on current master and make sure your "os-autoinst" subfolder is up to date to use an updated style check. You may have to create a symlink to `.perltidyrc` manually, e.g. with `ln -s os-autoinst/.perltidyrc`

**Pull request closed without merging**